tests/units/Manager/DemandeClinique/ReponseManager.php

```diff
@@ -18,6 +24,12 @@ public function beforeTestMethod($testMethod)
 
         $this->reponseValidator = new \mock\App\Validator\DemandeClinique\ReponseValidator();
         $this->reponseValidator->getMockController()->valider = null;
+
+        // Création du mock pour ManagerRegistry
+        $managerRegistry = new \mock\Doctrine\Persistence\ManagerRegistry();
+
+        // Création du mock pour ReponseRepository en passant ManagerRegistry et ClassMetadata
+        $this->reponseRepository = new \mock\App\Repository\DemandeClinique\ReponseRepository($managerRegistry, $this->entityManagerInterface);
```

Comment:
J'ai joué un peu avec atoum mais je ne suis pas parvenu à avoir `$this->reponseRepository` bien mocké, ce qui m'a empêché d'utiliser `getTestedInstance()` qui attendait en nouveau paramètre un mock de `ReponseRepository`. Je n'ai donc pas pu terminer de fixer les tests existants et d'ajouter mes nouveaux tests autour de la validation